src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -196,6 +199,28 @@ public String getString()
         return value == null ? defaultVal : STRING_CONVERTER.convert(value);
     }
 
+    /**
```

Comment:
Pulled from https://issues.apache.org/jira/browse/CASSANDRA-16213, was going to add them there but need them in this patch as well, so copy/pasted from there (but added docs for the ones missing)